client/reader/search-stream/index.jsx

```diff
@@ -206,6 +206,30 @@ const SearchStream = React.createClass( {
 		return SearchCardAdapter( isRecommendations );
 	},
 
+	handleStreamMounted( c ) {
+		this.streamRef = c;
+	},
+
+	handleSearchBoxMounted( c ) {
+		this.searchBoxRef = c;
+	},
+
+	resizeSearchBox() {
```

Comment:
Probably want to use lodash `debounce` here to stop this happening too often. For example, in Manage Following we use:

```handleResize: debounce( function() {
  this.setState( { windowWidth: this.getWindowWidth() } );
}, 500 )```